pkg/util/mount/mount.go

```diff
@@ -33,6 +33,14 @@ const (
 	MountsInGlobalPDPath = "mounts"
 )
 
+type MountPathType string
+
+const (
+	mountPathSocket   MountPathType = "Socket"
```

Comment:
These values are copied from [`HostPathType`](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L738-L743).

In order to avoid import cycle, we should find a place to declare these const vars. @jsafrane WDYT?

Reply:
These values should be declared here and HostPath plugin should import them.